### PR TITLE
Move API Services into their Own Directories

### PR DESCRIPTION
**Pull request by evanugarte**
## What does this do?
to better match how our backend is structure the `api/` directory was reorganized. see backend structure in the photo below
image

## File structure
Below we can see each folder has its own `routes` and maybe `models` directory. Notice how `google_api/` has a special `email_templates/` directory.
```
├── config/
├── util/
├── logging_api
│   ├── routes/
│   ├── models/
│   └── server.js
├── google_api
│   ├── email_templates/
│   ├── routes/
│   └── server.js
└── main_endpoints
    ├── routes/
    ├── models/
    ├── util/
    └── server.js
```
## the `util/` and `config/` directory
there is a single `util/` directory outside of these 3 folders, which contain helpful functions used across the three folders. `config/` has keys n stuff.

### CODE REVIEW

File: api/google_api/routes/mailer.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 const express = require('express');
 const router = express.Router();
-const { SceGoogleApiHandler } = require('./SceGoogleApiHandler');
-const verification = require('./templates/verification');
-const { validateVerificationEmail, generateHashedId } = require('./auth');
+const { SceGoogleApiHandler } = require('../util/SceGoogleApiHandler');
+const verification = require('../email_templates/verification');
+// const { validateVerificationEmail } = require('../util/auth');
```
Comment (evanugarte) [inline api/google_api/routes/mailer.js:5]: commented this out as we will resolve this auth issue with @law-nguyen tomorrow

File: api/main_endpoints/routes/3DPrintingForm.js [hunk 2]
```diff
@@ -3,8 +3,7 @@
 const express = require('express');
 const router = express.Router();
 const PrintingForm3D = require('../models/PrintingForm3D.js');
-const settings = require('../util/settings');
-const logger = require(`${settings.util}/logger`);
```
Comment (seemavora) [inline api/main_endpoints/routes/3DPrintingForm.js:7]: just wondering, why was this removed?

File: api/package.json [hunk 3]
```diff
@@ -4,7 +4,10 @@
   "description": "contains all the APIs used by the SCE website",
   "main": "SceHttpServer.js",
   "scripts": {
-    "start": "node SceHttpServer.js",
+    "start": "nodemon util/SceHttpServer.js",
```
Comment (seemavora) [inline api/package.json:7]: Are we still planning on creating one package.json file for all the different subdivisions?

Comment (evanugarte) [reply api/package.json:7]: We can, it is creates some really annoying errors. For now we can keep it like this and when #356 is finished by @SOMLpj we can give it a shot

File: api/main_endpoints/routes/3DPrintingForm.js (hunk 2 above)
Comment (evanugarte) [reply api/main_endpoints/routes/3DPrintingForm.js:7]: `logger` is a file from a long time ago which would write limited logs to a file in Core-v4. It isn't used very much and that's why i took it out

**Comment by jerrylee17** [comment]
Rebase
